pyflakes/messages.py

```diff
@@ -116,6 +116,22 @@ def __init__(self, filename, loc, name):
         self.message_args = (name,)
 
 
+class DuplicateDictionaryKey(Message):
+    message = 'dictionary key %s duplicated'
```

Comment:
I have a slight preference for "duplicated" being "repeated" everywhere, or at least in these messages, as that is how cpython describes it.

```bash
$ python -c "def foo(a, b): pass; foo(a=1, a=2)"
  File "<string>", line 1
SyntaxError: keyword argument repeated
```

But also "repeated" is much more accessible for ESL people.

"This word was duplicated." has a Flesch reading ease score of 54.7 (and is roughly ~US school grade 10) , whereas "This word was repeated." is 75.9 (~ grade 6)